pkg/proto/nbcontract/v1/networkconfig.proto

```diff
@@ -11,21 +10,21 @@ message NetworkConfig {
 }
 
 enum NetworkModeType {
-  NETWORK_MODE_UNSPECIFIED = 0;
-  NETWORK_MODE_BRIDGE = 1;
-  NETWORK_MODE_TRANSPARENT = 2;
+  NM_UNSPECIFIED = 0;
+  BRIDGE = 1;
+  TRANSPARENT = 2;
 }
 
 enum NetworkPluginType {
-  NETWORK_PLUGIN_TYPE_UNSPECIFIED = 0;
-  NETWORK_PLUGIN_TYPE_NONE = 1;
-  NETWORK_PLUGIN_TYPE_AZURE = 2;
-  NETWORK_PLUGIN_TYPE_KUBENET = 3;
+  NPT_UNSPECIFIED = 0;
```

Comment:
is there a reason we are shortening all of the enum names? i feel like it makes the code less readable

Reply:
Please see my response in another comment :)